sound/soc/sof/ipc4.c

```diff
@@ -388,7 +388,7 @@ static int sof_ipc4_tx_msg(struct snd_sof_dev *sdev, void *msg_data, size_t msg_
 
 	mutex_unlock(&ipc->tx_mutex);
 
-#ifdef DEBUG_VERBOSE
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_VERBOSE_IPC)
```

Comment:
@plbossart, I think the reason was that the payload dump makes the log unreadable and if we enable it with CONFIG_SND_SOC_SOF_DEBUG_VERBOSE_IPC then the CI logs will be hard to read (and long).
If I recall we concluded in #3443 that it should be not enabled by default but keep it guarded by broader DEBUG_VERBOSE.

I'm still on that side, or we should add a new SOF KCONFIG option to dump the message payload (or a flag in sof_debug) and do this for IPC3 and IPC4?